tedana/decomposition/pca.py

```diff
@@ -243,38 +244,74 @@ def tedpca(data_cat, data_oc, combmode, mask, adaptive_mask, t2sG,
     comptable['normalized variance explained'] = varex_norm
 
     # write component maps to 4D image
-    comp_ts_z = stats.zscore(comp_ts, axis=0)
```

Comment:
Why get rid of z-scoring? Unless it's somewhere else and I missed it?

Reply:
`computefeats2` handles z-scoring internally. Hopefully when #644 is merged it'll even do it correctly.